balrogscript/script.py

```diff
@@ -16,14 +16,35 @@
 
 
 def create_submitter(e, balrog_auth, config):
-    from balrog.submitter.cli import NightlySubmitterV4  # noqa: E402
+    from balrog.submitter.cli import NightlySubmitterV4, ReleaseSubmitterV4  # noqa: E402
     auth = balrog_auth
 
     if "tc_release" in e:
-        raise NotImplementedError("This logic piece has yet to be implemented")
+        log.info("Taskcluster Release style Balrog submission")
+
+        complete_info = e['completeInfo']
```

Comment:
I'm not sure if we always submit completes. I'd make them optional as well. Just add a check to make sure that either of them is set, aka `any([complete_info, partial_info])`.

Reply:
We always do  - https://github.com/mozilla-releng/beetmoverscript/blob/master/beetmoverscript/script.py#L274